index.js

```diff
@@ -115,6 +118,32 @@ cm.Item({
   }
 });
 
+cm.Item({
+  label: 'Send to mini player',
+  context: cm.SelectorContext('[href*="vine.co/v/"]'),
+  contentScript: 'self.on("click", function (node, data) {' +
+                 '  self.postMessage(node.href);' +
+                 '});',
+  onMessage: function(url) {
+    const id = getVineId(url);
+    updatePanel({domain: 'vine.co', id: id, src: ''});
+    getVineUrl(url, function(err, streamUrl) {
+      if (!err) updatePanel({src: streamUrl});
+    });
+  }
+});
+
+function getVineId(str) {
```

Comment:
I just did a `$ npm i get-video-id@latest` in a blank directory and got a `get-video-id@1.0.0` with all the Vine code in the README.